docs/wiki/framework/parachute-framework.md

```diff
@@ -16,9 +16,21 @@ version:
 
 ```cpp
 class CfgVehicles {
-    class BananaParachute {
+    class ParachuteBase;
+    class BananaParachute: ParachuteBase {
         ace_hasReserveParachute = 1;  // Add reserve parachute (1-enabled, 0-disabled)
         ace_reserveParachute = "ACE_ReserveParachute";  // Classname of the reserve parachute
     };
 };
 ```
+
+## 2. Adding failure cut delay
+
+```cpp
+class CfgVehicles {
+    class ParachuteBase;
+    class BananaParachute: ParachuteBase {
+        ace_parachute_failureDelay = 2;  // Add delay before parachute fails (time in seconds)
```

Comment:
Since you're using `getNumber`, the token can also be a simple expression string:

```cpp
        ace_parachute_failureDelay = "1 + random 2";
```
This will be evaluated every time the `getNumber` command is used.

Reply:
(because there is no automatic type conversion in Arma :^)

Reply:
Cool, did not know that was possible. The value however is intentionally set to `2`, it's the point air starts getting under the main parachute (which I would think could cause a rope to fail or something).